<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ITSL-LT52" failures="2" tests="6" name="appseConnect Test" time="90.744" errors="0" timestamp="2022-05-23T22:04:22 IST">
  <testcase classname="com.appseConnect.testCases.TC_LoginTest_001" name="loginTest" time="17.583"/>
  <testcase classname="com.appseConnect.testCases.TC_LoginTestDDT_002" name="loginTestDDT" time="20.402"/>
  <testcase classname="com.appseConnect.testCases.TC_LoginTestDDT_002" name="loginTestDDT" time="4.14">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.appseConnect.testCases.TC_LoginTestDDT_002.loginTestDDT(TC_LoginTestDDT_002.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTestDDT -->
  <testcase classname="com.appseConnect.testCases.TC_LoginTestDDT_002" name="loginTestDDT" time="9.209"/>
  <testcase classname="com.appseConnect.testCases.TC_LoginTestDDT_002" name="loginTestDDT" time="10.027"/>
  <testcase classname="com.appseConnect.testCases.TC_LoginTestDDT_002" name="loginTestDDT" time="3.083">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.appseConnect.testCases.TC_LoginTestDDT_002.loginTestDDT(TC_LoginTestDDT_002.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTestDDT -->
</testsuite> <!-- appseConnect Test -->
